### ISSUE TITLE
Setting additionalTasks means running "blendid" by itself won't enter watch mode

### ISSUE THREAD

**Issue by toolness** [vigetlabs/blendid#477]
I was following the instructions for [`additionalTasks`](https://github.com/vigetlabs/blendid#additionaltasks) and used code almost identical to the sample code in that section, but noticed that when I next ran `blendid`, while it did execute my custom tasks, it just exited after building the site, instead of starting a server and watching files for changes.

I *think* I was able to resolve this by adding `'watch'` to the end of my `additionalTasks.development.prebuild` array, so it might be worth documenting in the readme.

**Comment by samhernandez** [comment]
I found that the example with empty arrays like

```
  development: {
    prebuild: ['createPngSprite'],
    postbuild: []
  },
```

where `postbuild` is empty, `postbuild` will halt the sequence of tasks. This is probably a function of `gulpSequence`. Blendid's task source looks like this.

```
gulpSequence('clean', prebuild, tasks.assetTasks, tasks.codeTasks, static, postbuild, 'watch', cb)
```

Removing the `postbuild: []` line fixed it for me. `postbuild: null` and `postbuild: false` also work.

I also noticed in the [gulpSequence](https://github.com/teambition/gulp-sequence#usage) usage docs that tasks within an array will run in parallel which is useful info as well.
